src/modules/launcher/PowerLauncher/MainWindow.xaml

```diff
@@ -27,109 +27,111 @@
     WindowStartupLocation="Manual"
     WindowStyle="None"
     mc:Ignorable="d">
-    <Grid x:Name="RootGrid" MouseDown="OnMouseDown">
-        <Grid.RowDefinitions>
-            <RowDefinition Height="Auto" />
-            <RowDefinition Height="Auto" />
-            <RowDefinition Height="*" MaxHeight="{Binding Results.MaxHeight}" />
-        </Grid.RowDefinitions>
-        <Border
-            x:Name="SearchBoxBorder"
-            Grid.Row="0"
-            Padding="12,4,12,3">
-            <local:LauncherControl x:Name="SearchBox" />
-        </Border>
-
-        <!--  Have to use a Grid instead of a StackPanel for scrolling to work?  -->
-        <Grid
-            x:Name="KeywordsOverviewGrid"
-            Grid.Row="1"
-            MaxHeight="{Binding Results.MaxHeight}"
-            Visibility="{Binding PluginsOverviewVisibility}">
+    <Border x:Name="MainBorder">

```

Comment:
```suggestion
    <Border x:Name="MainBorder">
    <Border x:Name="MainBorder" BorderBrush="{DynamicResource {x:Static SystemColors.ActiveBorderBrushKey}}">
```

Reply:
There, I updated it to a single line.

Reply:
This suggestion requires that we remove `MainBorder.BorderBrush = System.Windows.Media.Brushes.Gray;` from MainWindow.xaml.cs as well, right?

Reply:
@mantaionut , can you please integrate and test this suggestion as well? Thank you!

Reply:
Yeah, sorry. Added the other suggestion to be clear.